src/bindings/go-elektra/.travis.yml

```diff
@@ -0,0 +1,13 @@
+dist: focal
```

Comment:
I think this file is pretty useless, as `.travis.yml` would need to be at the root directory of the repository.